egs2/voxceleb/spk1/local/data.sh

```diff
@@ -140,9 +140,14 @@ fi
 
 if [ ${stage} -le 4 ] && [ ${stop_stage} -ge 4 ]; then
     echo "Change into kaldi-style feature."
-    mkdir -p ${trg_dir}/voxceleb1_test
-    mkdir -p ${trg_dir}/voxceleb2_dev
-    python local/data_prep.py --src "${data_dir_prefix}/voxceleb1/test" --dst "${trg_dir}/voxceleb1_test"
-    python local/data_prep.py --src "${data_dir_prefix}/voxceleb2/dev" --dst "${trg_dir}/voxceleb2_dev"
+    #mkdir -p ${trg_dir}/voxceleb1_test
+    #mkdir -p ${trg_dir}/voxceleb2_dev
+    #python local/data_prep.py --src "${data_dir_prefix}/voxceleb1/test" --dst "${trg_dir}/voxceleb1_test"
+    #python local/data_prep.py --src "${data_dir_prefix}/voxceleb2/dev" --dst "${trg_dir}/voxceleb2_dev"
```

Comment:
Please deal with commented lines (remove lines or remove `#`)

Reply:
Oh, sorry - this was only for me to skip locally. I will uncomment it (remove `#`)